src/theme/theme.ts

```diff
@@ -226,8 +229,8 @@ const theme = createTheme({
       fontFamily: "Raleway",
       fontStyle: "normal",
       fontWeight: 800,
-      fontSize: "80px",
-      lineHeight: "96px",
+      fontSize: "32px",
+      lineHeight: "37.57px",
```

Comment:
This is a big difference in values for all H1 variants.... now the h1 and h2 have the same size. Was this a change requested by design? 🤔